docs/user/integration/loki/README.md

```diff
@@ -0,0 +1,243 @@
+# Integrate with Loki
+
+## Overview
+
+The following instructions outline how to use [Loki](https://github.com/grafana/loki/tree/main/production/helm/loki) as a logging backend with Kyma's [LogPipeline](../../02-logs.md) or with [Promtail](https://grafana.com/docs/loki/latest/clients/promtail/).
+
+>**CAUTION:** This guide uses the Grafana Loki version, which is distributed under AGPL-3.0 only and might not be free of charge for commercial usage.
+
+## Table of Content
+
+- [Prerequisites](#prerequisites)
+- [Preparation](#preparation)
+- [Loki Installation](#loki-installation)
+- [Log agent installation](#log-agent-installation)
+- [Grafana installation](#grafana-installation)
+- [Grafana Exposure](#grafana-exposure)
+
+## Prerequisites
+
+- Kyma as the target deployment environment
```

Comment:
Same as with Jaeger

Reply:
I wasn't sure what you mean [there](https://github.com/kyma-project/telemetry-manager/pull/586/files#r1410973955) either. Please explain.

Reply:
I tried now